addons/website_sale/controllers/main.py

```diff
@@ -731,6 +731,7 @@ def cart(self, access_token=None, revive='', **post):
             'date': fields.Date.today(),
             'suggested_products': [],
         })
+        values.update(order._get_website_sale_extra_values())
         if order:
```

Comment:
```suggestion
        if order:
            values.update(order._get_website_sale_extra_values())
```
I believe there is no need to add any loyalty related values if there is no order at all. (and the place where the template adds the messages is only shown if there is an order anyway)